test/e2e/webhook-http-proxy-test

```diff
@@ -144,10 +157,10 @@ for i in `seq 1 $TAINT_CHECK_CYCLES`; do
   if [[ $evicted -eq 1 && $sent -eq 0 ]] && kubectl logs $pod_id -n kube-system | grep 'Webhook Success' >/dev/null; then
     echo "✅ Verified that webhook successfully sent"
     sent=1
-    squid_worker_pods=$(kubectl get pods -o json | jq '.items[] | select( .metadata.name | contains("squid") ) | .metadata.name as $name | select( .spec.nodeName | contains("worker") ) | .spec.nodeName as $nodename | $name' -r)
   fi
 
-  if [[ $sent -eq 1 ]] && kubectl exec -it $(echo $squid_worker_pods | cut -d' ' -f1) -- cat /var/log/squid/access.log | grep "${WEBHOOK_URL}" >/dev/null; then
+  webhook_hostname=$(echo "${WEBHOOK_URL}" |sed -e 's|^[^/]*//||' -e 's|/.*$||')
```

Comment:
This is super complain-y, but the use of the pipe character in the sub-shell *and* the sed expressions is kinda hard to read.

```suggestion
  webhook_hostname=$(echo "${WEBHOOK_URL}" | sed -e 's@^[^/]*//@@' -e 's@/.*$@@')
```

Reply:
that's fair 👍